File: textmagic-rest-csharp/TextmagicRest/Model/UnsubscribedContactsResult.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using RestSharp.Deserializers;

namespace TextmagicRest.Model
{
    /// <summary>
    /// List of UnsubscribedContact objects
    /// </summary>
    public class UnsubscribedContactsResult
    {
        [DeserializeAs(Name = "resources")]
        public List<UnsubscribedContact> UnsubscribedContacts { get; set; }
    }
}

File: textmagic-rest-vbnet/TextmagicRest.ConvertedToVBNet/Model/UnsubscribedContactsResult.vb
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports RestSharp.Deserializers

Namespace Model
	''' <summary>
	''' List of UnsubscribedContact objects
	''' </summary>
	Public Class UnsubscribedContactsResult
		<DeserializeAs(Name := "resources")> _
		Public Property UnsubscribedContacts() As List(Of UnsubscribedContact)
			Get
				Return m_UnsubscribedContacts
			End Get
			Set
				m_UnsubscribedContacts = Value
			End Set
		End Property
		Private m_UnsubscribedContacts As List(Of UnsubscribedContact)
	End Class
End Namespace
